presto-product-tests/bin/product-tests-suite-5.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-hive-impersonation \
```

Comment:
According to https://github.com/prestosql/presto/blob/c808deea818325b300cb3d185ef1af0f7a056cd8/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java#L144
> this.impersonationEnabled = thriftConfig.isImpersonationEnabled() && hiveMetastoreAuthentication instanceof KerberosHiveMetastoreAuthentication;

the impersonation is supported only when kerberos is enabled
i think with setUGI, we should support impersonation with and without kerberos

-- but this also signals we don't have a direct test for an impersonation, something that would fail when impersonation does not work

Reply:
I think this should be visible if you add `authorization` here. (It should be included here anyway, it contains TestRoles, this is important to run here)